.gitlab-ci.yml

```diff
@@ -62,3 +62,29 @@ build_wheel:
     # create the wheel and upload to pypi
     - sh scripts/create_assembl_wheel.sh
     - twine upload -u bluenove --skip-existing dist/*
+
+deploy_to_dev:
+  stage: tmp_master_for_ci_cd
+  only:
+    - master
```

Comment:
stage: deploy_to_dev
only: tmp_master_for_ci_cd
???

Reply:
Oh yeah I misunderstood @cedricmessiant 's comment :) I wasn't fully awake I think

Reply:
mater and tmp_master_for_ci_cd ?

Reply:
Well yes isn't it what you said in your review @cedricmessiant ? Or should it be just the tmp branch?